CMakeLists.txt

```diff
@@ -190,6 +203,9 @@ if(OCOS_ENABLE_PYTHON)
     message(FATAL_ERROR "Python3_FIND_REGISTRY is not NEVER")
   endif()
   find_package(Python3 COMPONENTS Interpreter Development.Module NumPy)
+  if (NOT Python3_FOUND)
```

Comment:
could missing lumpy also trigger this error?

Reply:
missing numpy will reminder user not founding numby and trigger this error, so I update the fatal error message.